lib/pymedphys/tests/dicom/test_coords.py

```diff
@@ -88,3 +91,41 @@ def test_extract_iec_fixed_xyz():
 @pytest.mark.pydicom
 def test_extract_dicom_patient_xyz():
     run_xyz_function_tests("DICOM")
+
+
+@pytest.mark.pydicom
+def test_coords_in_datasets_are_equal():
+    test_dicom_dict = {
+        "ImagePositionPatient": [-1.0, -1.0, -1.0],
+        "ImageOrientationPatient": [1, 0, 0, 0, 1, 0],
+        "BitsAllocated": 32,
+        "Rows": 3,
+        "Columns": 3,
+        "PixelRepresentation": 0,
+        "SamplesPerPixel": 1,
+        "PhotometricInterpretation": "MONOCHROME2",
+        "PixelSpacing": [1.0, 1.0],
+        "GridFrameOffsetVector": [0.0, 1.0, 2.0],
+        "PixelData": np.ones((3, 3, 3)).tobytes(),
+    }
+
+    ds1 = dicom_dataset_from_dict(test_dicom_dict, ensure_file_meta=True)
+    ds2 = copy.deepcopy(ds1)
+    assert coords_in_datasets_are_equal([ds1, ds2])
+
+    # y-shift (for DICOM HFS)
+    ds2.ImagePositionPatient = [-1.0, -1.1, -1.0]
+    assert not coords_in_datasets_are_equal([ds1, ds2])
```

Comment:
Simple and beautiful :slightly_smiling_face:.

Reply:
🙂